src/main/java/edu/usf/cutr/gtfsrtvalidator/api/resource/GtfsFeed.java

```diff
@@ -221,8 +224,9 @@ private String getSaveFilePath(String gtfsFeedUrl, HttpURLConnection connection)
         } else {
             //Extracts file name from header field
             int index = disposition.indexOf("filename=");
+            int filenameLastIndex = disposition.indexOf("\"", index+10);
```

Comment:
Ah, right. You're correct.  So this is Ok as is, although it would still be nice to have unit tests for loading GTFS.   If this is straight forward to implement please add it, if not we can live without it for now.